lib/std/crypto/sha3.zig

```diff
@@ -505,18 +499,26 @@ fn TupleHashLike(comptime security_level: u11, comptime default_delim: u8, compt
     };
 }
 
-/// The NIST encoded length format.
-pub const NistLengthEncoding = enum { left, right };
+/// The NIST SP 800-185 encoded length format.
+pub const NistLengthEncoding = enum {
+    left,
+    right,
 
-/// A length encoded according to NIST SP 800-185.
-pub const NistEncodedLength = struct {
-    /// The size of the encoded value, in bytes.
-    len: usize = 0,
-    /// A buffer to store the encoded length.
-    buf: [@sizeOf(usize) + 1]u8 = undefined,
+    /// A length encoded according to NIST SP 800-185.
+    pub const NistEncodedLength = struct {
```

Comment:
https://ziglang.org/documentation/master/#Avoid-Redundant-Names-in-Fully-Qualified-Namespaces

can you come up with a better name than `std.crypto.sha3.NistLengthEncoding.NistEncodedLength`?

Reply:
`NistEncodedLength` has been renamed to `Length`.

Reply:
Works for me!

Reply:
🎉